examples/cifar10/predict.py

```diff
@@ -52,7 +52,7 @@ def predict(net, images, dev, topk=5):
 def load_dataset(filepath):
     print('Loading data file %s' % filepath)
     with open(filepath, 'rb') as fd:
-        cifar10 = pickle.load(fd)
+        cifar10 = pickle.load(fd, encoding='latin1')
```

Comment:
Yes, we need latin1 for loading the files in cifar10 example. See :

https://stackoverflow.com/questions/28218466/unpickling-a-python-2-object-with-python-3